crates/storage/src/column.rs

```diff
@@ -71,6 +71,10 @@ pub enum Column {
     // TODO: Remove this column and use `Metadata` column instead.
     /// Table for genesis state import progress tracking.
     GenesisMetadata = 21,
+
+    /// See [`FullFuelBlocks`](crate::tables::FullFuelBlocks)
+    /// we don't use this table at the moment, only used for benchmarks
+    FullFuelBlocks = 22,
```

Comment:
[nit]: If we're gonna keep it, maybe put it above `GenesisMetadata`.

Reply:
`GenesisMetadata` is 21, so logically this should come after, right? or is it the general preference to keep everything before the `GenesisMetadata` table?

Reply:
Yeah. If you look at previous additions we've always kept `GenesisMetadata` on the bottom.